doc/lua_api.txt

```diff
@@ -2615,6 +2615,10 @@ Call these functions only at load time!
 * `minetest.register_on_leaveplayer(func(ObjectRef, timed_out))`
     * Called when a player leaves the game
     * `timed_out`: True for timeout, false for other reasons.
+* `minetest.register_on_auth_fail(func(ip, name))`
```

Comment:
Please use `name, ip` as in `minetest.register_on_prejoinplayer(func(name, ip))`

Reply:
done